src/test/java/jalau/at18/katas/phonenumber/sarai/MainTest.java

```diff
@@ -0,0 +1,10 @@
+package jalau.at18.katas.phonenumber.sarai;
+
+import static org.junit.Assert.*;
+public class MainTest {
+    public void shouldReturnPhoneNumbers() {
+        String input = "1 2 3 4 5 6 7 8 9 0";
+        String expected = "(123)456-7890";
```

Comment:
Almost, kata requirement expects that result contains a space between ")" and the next number
```
createPhoneNumber([1, 2, 3, 4, 5, 6, 7, 8, 9, 0]) // => returns "(123) 456-7890"
```